README.md

```diff
@@ -13,13 +13,147 @@ I wrote this module because I couldn't find one that could be simply plugged int
 
 ## Getting started
 
+If you have your own react-native project now, 
+
+install and start using react-native-qrcode-scanner:
+
+```sh
+npm install react-native-qrcode-scanner
+
+react-native link react-native-qrcode-scanner
+```
+
+If you want to use this from sample, make clear that `react` and `react-native` is installed. (Doesn't matter if Globally or Locally)
```

Comment:
I think this is a good addition, but we should move this up to the start (before installing the library) so that the order of installing items is more logical